tests/beignet/nn/functional/test__torsion_angle_loss.py

```diff
@@ -0,0 +1,17 @@
+import beignet.nn.functional
+import torch
+
+
+def test_torsion_angle_loss():
```

Comment:
@kleinhenz Just a smoke test. I will revisit with improved unit tests after I implement the scaffolding for all the losses.